SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -25,6 +25,9 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	TSubclassOf<USpatialMultiWorkerSettings> GetMultiWorkerSettingsClass(bool bForceNonEditorSettings = false);
 
 #if WITH_EDITORONLY_DATA
+	UPROPERTY(EditAnywhere, Category = "Testing", Meta = (FilePathFilter = "ini"))
+	FFilePath SettingOverrides;
```

Comment:
most uber of nits, but should this be `SettingsOverride` ?

Reply:
Sure